test/fixtures/output/kafka/partition/describe-onprem.golden

```diff
@@ -0,0 +1,6 @@
++--------------+-----------+
+| Cluster      | cluster-1 |
```

Comment:
We could consider changing this to "Cluster Name" in v4?

Reply:
It's an id, not a name. So we should either keep it as is or add `ID` in v4.

Reply:
I don't think I have a strong preference here

Reply:
Maybe we can just make all of them say `Name` or `ID` (for the resources where they aren't the same thing), to remove potential ambiguity.

Reply:
I think we removed "ID" from a lot of them a while ago to save horizontal space